spec.html

```diff
@@ -3042,6 +3095,25 @@ <h1>CheckBoundNames ( _expectedBound_, _actualBound_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-checkinnerlazyfunctions" aoid="CheckInnerLazyFunctions">
+      <h1>CheckInnerLazyFunctions ( _funcNode_ )</h1>
+      <emu-alg>
+          1. Assert: _funcNode_ is a `LazyFunctionDeclaration`, a `LazyFunctionExpression`, a `LazyMethod`, a `LazyGetter`, a `LazySetter`, or a `LazyArrowExpression`.
+          1. NOTE: All asserted immediately inner (i.e. not nested within another inner function) lazy functions in `innerLazyFunctions` must be found.
+          1. Let _innerFunctions_ be a new empty List.
+          1. For each _link_ in _funcNode_`.innerLazyFunctions`, do
+              1. Let _linkedNode_ be the node linked to by _link_.
+              1. NOTE: The above step is up to the surface encoding.
+              1. Assert: _linkedNode_ is a `LinkableFunctionDeclaration`, a `LinkableFunctionExpression`, a `LinkableMethod`, a `LinkableGetter`, a `LinkableSetter`, or a `LinkableArrowExpression`.
+              1. Add _linkedNode_ as the last element of _innerFunctions_.
```

Comment:
shouldn't we put `.fun` attribute etc for each?
otherwise _candidateInnerFunction_ below won't match any.

Reply:
Good catch.